codegen-sbt/src/sbt-test/compiletime-codegen/test-compile/build.sbt

```diff
@@ -34,8 +34,8 @@ lazy val calibanLib: Seq[ModuleID] =
   }
 
 lazy val sttp = Seq(
-  "com.softwaremill.sttp.client3" %% "core" % "3.10.3",
-  "com.softwaremill.sttp.client3" %% "zio"  % "3.10.3"
+  "com.softwaremill.sttp.client4" %% "core" % "4.0.0",
+  "com.softwaremill.sttp.client4" %% "zio"  % "4.0.0"
```

Comment:
Probably not necessary, but should these be set to 4.0.2 as well?

Reply:
Done